app/src/common/img/ic-invite-inline.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="16"><path d="M8 9H0V3h8v6zM7 4L4 6.158 1 4v1.752L4 8l3-2.248V4zm6.02 3.868a3.489 3.489 0 0 1-3.49-3.49 3.489 3.489 0 0 1 3.49-3.49 3.489 3.489 0 0 1 3.49 3.49 3.489 3.489 0 0 1-3.49 3.49zm-6.98 5.234c0-2.32 4.651-3.49 6.98-3.49 2.33 0 6.98 1.17 6.98 3.49v1.918H6.04v-1.918z" fill="#1A9CB0" fill-rule="evenodd"/></svg>
```

Comment:
We have this icon. `app/src/pages/inside/membersPage/membersPageToolbar/img/ic_invite-inline.svg`
Please move all icons from that folder to `common/img`.